sklearn/manifold/t_sne.py

```diff
@@ -83,21 +87,27 @@ def _joint_probabilities_nn(distances, neighbors, desired_perplexity, verbose):
 
     Returns
     -------
-    P : array, shape (n_samples * (n_samples-1) / 2,)
-        Condensed joint probability matrix.
+    P : csr sparse matrix, shape (n_samples, n_samples)
+        Condensed joint probability matrix with only nearest neighbors.
```

Comment:
Are both half (above diagonal and below diagonal) of the array useful, or it is only one? In the latter case, it would be useful to indicate it.